src/store/commit.js

```diff
@@ -0,0 +1,33 @@
+import { indexGenerator } from "../api/fetch";
```

Comment:
The name of this file doesn't feel right to me. Something like `actions` maybe?

Reply:
I meant the name of `src/store/commit.js`, not `src/api/fetch.js`. Sorry for the confusion.